src/display/editor/freetext.js

```diff
@@ -139,9 +139,7 @@ class FreeTextEditor extends AnnotationEditor {
 
   /** @inheritdoc */
   static initialize(l10n, uiManager) {
-    AnnotationEditor.initialize(l10n, uiManager, {
-      strings: ["pdfjs-free-text-default-content"],
```

Comment:
It looks like this was the only occurrences that passed in `strings`, or an `options` parameter for that matter to `AnnotationEditor.initialize`, so unless I'm mistaken I think we should also be able to remove https://github.com/mozilla/pdf.js/blob/master/src/display/editor/editor.js#L239-L243 and the entire `options` argument there to simplify things even further.

Reply:
Good catch!